web/app/js/components/TopRoutes.jsx

```diff
@@ -62,24 +61,23 @@ class TopRoutes extends React.Component {
       PrefixedLink: PropTypes.func.isRequired,
     }).isRequired,
     classes: PropTypes.shape({}).isRequired,
-    query: topRoutesQueryPropType,
+    query: topRoutesQueryPropType.isRequired,
+    setQuery: PropTypes.func.isRequired,
   }
-  static defaultProps = {
```

Comment:
The injected `query` prop is replacing the defaultProps to an object with undefined values (if query string value is not defined of course)